src/main/java/com/google/sps/api/tasks/TasksProvider.java

```diff
@@ -14,9 +14,34 @@
 
 package com.google.sps.api.tasks;
 
+import com.google.api.services.tasks.model.Task;
 import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class TasksProvider {
-  public static final ImmutableList<String> TASKS_SAMPLE = ImmutableList.of(
+  public static final ImmutableList<String> TASKS_SAMPLE_TITLES = ImmutableList.of(
       "Review code", "Write the design doc", "Talk to PM", "Investigate report", "Prepare slides");
+
+
+  private final List<Task> TASK_MODEL_LIST = new ArrayList<Task>();
```

Comment:
nit: non constants (i.e. not static) fields should have camelCase names and not CONSTANT_CASE.

Reply:
Note that IntelliJ has an automatic refactoring fo rthis rename that will update all classes (including your tests)